Add atype tx to view

howeik opened the pull request:
Fixes #4 

Text answer type show's a textbox now.
Tested that putting input in, clicking next, then clicking back preserves the input.

rtibbles commented on survey/templates/survey/questiondetail.html:
```diff
@@ -165,7 +165,9 @@ <h3 align="center"> {{ page.page_title }} </h3>
                                 <textarea rows="5" cols="60" name="{{ q.id }}_{{ answer.id }}" id="{{ answer.id }}" value="{{ resp }}">{{ answer.atext }}</textarea><br />
-                            
+                            {% elif answer.atype == 'Tx' %}
+                                <label class="answer">
```
Unclosed label tag.

Might want to just use an <input></input> tag rather than the XML style <input />, and wrap the answer text in that.

howeik replied:
closed the label tag and made the input tag not self close

howeik commented:
went ahead and did the html cleanup for all answer types. also removed the <br> that was at the end of every answer.atype case and just added one <br> at the end of every answer. see: 16bb8ed
